webapp/src/note_rhs.tsx

```diff
@@ -0,0 +1,179 @@
+import React, {FC, useEffect, useState} from 'react';
+
+import {getCurrentChannelId} from 'mattermost-redux/selectors/entities/common';
+import {GlobalState} from 'mattermost-redux/types/store';
+import {useDispatch, useSelector} from 'react-redux';
+
+import styled from 'styled-components';
+
+import {updateUpstream, fetchNote} from 'client';
+import {makeSelectNoteValue, setNoteValue} from 'redux_connectors';
+
+// @ts-ignore
```

Comment:
What are we ignoring here?

Reply:
eslint tells us not to use the `window` object :)